src/bin/pg_autoctl/monitor.c

```diff
@@ -1641,7 +1641,7 @@ monitor_print_state(Monitor *monitor, char *formation, int group)
 		{
 			sql =
 				"SELECT * FROM pgautofailover.current_state($1) "
-				"ORDER BY node_id";
+				"ORDER BY nodename";
```

Comment:
I don't think that's a good idea, the node_id is stable, the nodename can change at any time.